src/DOM/Event/EventTarget.purs

```diff
@@ -1,44 +1,39 @@
 module DOM.Event.EventTarget where
 
 import Prelude
-import Control.Monad.Eff (kind Effect, Eff)
-import Control.Monad.Eff.Exception (EXCEPTION)
-import DOM (DOM)
+import Control.Monad.Effect (Effect)
 import DOM.Event.Types (EventTarget, Event, EventType)
 
 -- | A boxed function that can be used as an event listener. This is necessary
--- | due to the underlying implementation of Eff functions.
-foreign import data EventListener :: # Effect -> Type
+-- | due to the underlying implementation of Effect functions.
+foreign import data EventListener :: Type
 
--- | Creates an EventListener from a normal PureScript Eff function.
+-- | Creates an EventListener from a normal PureScript Effect function.
 foreign import eventListener
-  :: forall eff a
-   . (Event -> Eff eff a)
-  -> EventListener eff
+  :: forall a
+   . (Event -> Effect a)
+  -> EventListener
```

Comment:
Since we're talking about related things on Slack just now, I remembered there's a breaking change I wanted to make here - `eventListener` should return in `Effect`, since it breaks referential transparency also.